.github/workflows/deploy-testflight.yml

```
@@ -698,16 +677,51 @@ jobs:
+          echo "🔍 PR 번호 추출 및 자동 병합 준비 중..."
+
           # workflow_run에서 PR 번호 추출
           PR_NUMBER=$(gh api repos/${{ github.repository }}/pulls \
             --jq '.[] | select(.head.sha == "${{ github.event.workflow_run.head_sha }}") | .number' \
             | head -1)
 
           if [ -n "$PR_NUMBER" ]; then
-            echo "🎉 모든 작업이 성공했습니다. PR #$PR_NUMBER을 병합합니다."
-            gh pr merge $PR_NUMBER --squash --admin
+            echo "📋 PR #$PR_NUMBER 정보 확인 중..."
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)